conda.recipe/build.sh

```diff
@@ -0,0 +1,36 @@
+#!/usr/bin/env bash
+
+python setup.py install --single-version-externally-managed --record=record.txt
+
+# Copy non-.py files
+
+# tethys_apps
+cp -r $SRC_DIR/tethys_apps/static $SP_DIR/tethys_apps/
```

Comment:
I think we ought to handle the resources through the setup.py so that it installs manually the same way it does in the conda environment. The app setup.py use a utility function for finding all the resource files. See: https://github.com/tethysplatform/tethys/blob/master/tethys_cli/scaffold_templates/app_templates/default/setup.py_tmpl